File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/Maxxiburger.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Food;

/* JADX INFO: loaded from: classes3.dex */
public class Maxxiburger extends Food {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Consumable, it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.food_maxxiburger_name;
        this.idDescription = R.string.food_maxxiburger_description;
        this.idImage = R.drawable.maxxiburger;
        this.price = 220L;
        this.feedPower = 3000;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/Maxxiburger.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Food

class Maxxiburger : Food() {
    override fun configureProperties() {
        idName = R.string.food_maxxiburger_name
        idDescription = R.string.food_maxxiburger_description
        idImage = R.drawable.maxxiburger
        price = 220L
        feedPower = 3000
    }
}
